docs/edge-authorizers/overview.mdx

```diff
@@ -13,6 +13,15 @@ instances running as close to your application as possible.
 
 <img src="/aserto-edge-authorizers.png" />
 
+## Edge Authorizers and Topaz
+
+Edge Authorizers are Topaz instances. What makes them different is how they are configured. Edge Authorizers connect to the Aserto Control Plane and light up with additional functionality:
+
+* Policy changes and data are pushed down when changes occur
```

Comment:
what about directory changes that are synced with all connected authorizers in near-real time. From my understanding it is the control plane that add a  high speed data fabric that pushes all the changes to policy or directory from the control plane to the topaz instances, but I might be wrong!

Reply:
Good catch, fixed.